about.html

```diff
@@ -18,6 +18,7 @@ <h2 id="rules">Rules <a href="#rules">¶</a></h2>
 <li>Automated posts should be <strong>unlisted</strong> (rendering visible to everybody, but not appearing on the local timeline) to keep the local timeline of our instance a place of community dialogue and human interaction. This extends to bots, feed posters, Twitter "retweets" and Twitter crossposts with broken mentions.</li>
 <li>Bots may only interact with a user when they're invited by that user to do so.</li>
 <li>If you need to post with a high frequency, please set your posts to be <strong>unlisted</strong> to keep the local timeline readable.</li>
+<li>In discussions, please remain civil, do not insult the people you're talking to, and try to avoid irony, sarcasm, or similar modes of languge – they typically don't translate well to written language and  tend to escalate discussions or missunderstandings.</li>
```

Comment:
U. a. zwei Leerzeichen vor tend entfernt:
<li>In discussions, please remain civil, do not insult the people you're talking to and try to avoid irony, sarcasm, or similar modes of language – they typically don't translate well to written language and tend to escalate discussions or missunderstandings.</li>

Reply:
This pull request has been merged and the discussion is hence over – if you want to add something, please open a pull request. (Though please note that this is HTML, so double spaces will not show up in any typical rendering.)